composer.json

```diff
@@ -6,6 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^8.1",
+        "filament/filament": "^3.0",
```

Comment:
Do we need this dependency?

Reply:
Nope this was a mistake, was playing with that in a different dir and must have ran it here too 🤦🏻